test/uk/gov/hmrc/softdrinksindustrylevy/connectors/ContactFrontendConnectorSpec.scala

```diff
@@ -17,35 +17,41 @@
 package uk.gov.hmrc.softdrinksindustrylevy.connectors
 
 import java.time.Instant
-import com.github.tomakehurst.wiremock.client.WireMock.{aResponse, post, stubFor, urlEqualTo, urlPathEqualTo}
-import uk.gov.hmrc.http.HeaderCarrier
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import play.api.libs.json.JsValue
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
 import uk.gov.hmrc.softdrinksindustrylevy.models.connectors._
-import scala.util.{Failure, Success, Try}
+import uk.gov.hmrc.softdrinksindustrylevy.util.FakeApplicationSpec
 
-class ContactFrontendConnectorSpec extends WiremockSpec {
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
 
-  object TestContactConnector extends ContactFrontendConnector(httpClient, environment.mode, configuration) {
-    override lazy val contactFrontendUrl: String = mockServerUrl
-  }
+class ContactFrontendConnectorSpec extends FakeApplicationSpec {
+
+  val connector = app.injector.instanceOf[ContactFrontendConnector]
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
+  val mockHttpClient = mock[HttpClient]
+
+  implicit lazy val ec: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
+
   "attempted contact form should fail if contact service is not available" in {
-    stubFor(
-      post(urlPathEqualTo("/contact/contact-hmrc/form?resubmitUrl=/"))
-        .willReturn(aResponse().withStatus(500)))
+    when(mockHttpClient.POST[JsValue, HttpResponse](any(), any(), any())(any(), any(), any(), any()))
+      .thenReturn(Future.failed(new Exception("")))
 
-    Try(TestContactConnector.raiseTicket(sub, "safeid1", Instant.now()).futureValue) match {
+    connector.raiseTicket(sub, "safeid1", Instant.now()) onComplete {
```

Comment:
keeping `.futureValue` is a good idea instead of using `onComplete` , as by the time onComplete run the test might have finished running